pkg/commands/execution_context.go

```diff
@@ -525,6 +529,15 @@ func (ctx *executionContext) completeTask(event operationEvent) {
 	ctx.Lock()
 	delete(ctx.running, event.task.taskID)
 	ctx.completed = append(ctx.completed, event.task)
+	failedTests := 0
+	for _, t := range ctx.completed {
+		if t.test.Status == model.StatusFailed {
+			failedTests++
+		}
+	}
```

Comment:
Why we need this loop? Can we just check `event.task.Status` and keep `failedTests` in context?

Reply:
Yes, as optimization, we can. I'll implement it, thanks.